src/libstd/sys/unix/process/process_common.rs

```diff
@@ -297,9 +297,14 @@ impl Stdio {
                 let mut opts = OpenOptions::new();
                 opts.read(readable);
                 opts.write(!readable);
+                #[cfg(not(target_os = "redox"))]
                 let path = unsafe {
                     CStr::from_ptr("/dev/null\0".as_ptr() as *const _)
                 };
+                #[cfg(target_os = "redox")]
+                let path = unsafe {
+                    CStr::from_ptr("null:\0".as_ptr() as *const _)
+                };
```

Comment:
Could this use `if cfg!(...)` if possible to avoid on using `#[cfg]`?

Reply:
Yep